packages/account/src/Components/financial-details/financial-details.tsx

```diff
@@ -59,6 +60,11 @@ const FinancialDetails = (props: TFinancialDetails) => {
         return errors;
     };
 
+    const fields_to_scroll_to_top = isMobile()
+        ? ['income_source', 'account_turnover', 'estimated_worth']
+        : ['income_source'];
+    const fields_to_scroll_to_end = isMobile() ? [] : ['account_turnover', 'estimated_worth'];
```

Comment:
```suggestion
    const fields_to_scroll_to_end = isMobile() ? [] : ['account_turnover', 'estimated_worth'];
```
maybe it's better to name it as `fields_to_scroll_to_bottom`?